tests/test_models.py

```diff
@@ -1,27 +1,87 @@
-#!/usr/bin/env python
```

Comment:
Add a test for `EnterpriseCustomer.all_objects` if you choose to continue filtering by default.